plugins/HHAnalyzer.cc

```diff
@@ -231,7 +203,8 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
             // Note that ID and isolation criteria are in both electron and muon loops
             if (!dilep.isOS)
                 continue;
-            if (event.isRealData() && !(leptons[ilep1].hlt_DR_matchedObject < m_hltDRCut && leptons[ilep2].hlt_DR_matchedObject < m_hltDRCut))
+            // Throw event if there is no matched trigger path
+            if (event.isRealData() && leptons[dilep.ilep1].hlt_idx == -1 && leptons[dilep.ilep2].hlt_idx == -1)
```

Comment:
I think by asking hlt_idx != -1 we were accepting events where both selected leptons actually triggered only leg 2 if e.g. a third object in the event had triggered leg1 (small effect I agree). Anyway, thanks for the change! Out of curiosity, do you have an estimate of the matching efficiency on e.g. DoubleMu? I just computed the one on 2015D (with the rough trigger matching) : ~98%